src/Paramore.Brighter/Message.cs

```diff
@@ -48,6 +55,11 @@ public class Message : IEquatable<Message>
         /// <value>The body.</value>
         public MessageBody Body { get; private set; }
 
+        /// <summary>
+        /// Gets the details for the Cloud Events specification
+        /// </summary>
+        public MessageCloudEvents CloudEvents { get; private set; }
```

Comment:
I think these are just part of MessageHeader, and CE is 'transport specific')

Reply:
Renamed and moved